modules/local/check_dependencies/resources/usr/bin/main.py

```diff
@@ -318,13 +312,13 @@ def update_relational_mapping(relational_mapping,analysis_types):
             if len(analysis_types[schema]['externalValidations'])==0:
                 continue
             if analysis_types[schema]['externalValidations'][0]['url'].endswith("health"):
-                analysis_types[schema]['externalValidations'][0]['url']="https://submission.pcgl-dev.cumulus.genomeinformatics.org/validator/category/1/entity/experiment/exists?organization={studyId}&value={value}"
+                analysis_types[schema]['externalValidations'][0]['url']="https://submission.pcgl-dev.cumulus.genomeinformatics.org/validator/entity/experiment/field/submitter_experiment_id/exists?study={study}&value={value}"
```

Comment:
Although the schema here is not really used, why do we hardcode to "https://submission.pcgl-dev.cumulus.genomeinformatics.org" instead of using `clinical_url`?

Reply:
Removed this line in : [remove unneeded line](https://github.com/Pan-Canadian-Genome-Library/molecular-data-submission-workflow/pull/43/commits/473037d0b2c5ec2513e82b9ada546954c6107081)

Originally it was needed to substitute out health endpoint for the validator endpoint that we use to retrieve dependent entity